draft-oid4vc-haip-sd-jwt-vc.md

```diff
@@ -186,37 +186,125 @@ In addition, this profile defines the following additional requirements.
+
+The following is a non-normative example of an authorization details object with Credential format `vc+sd-jwt`.
+
+<{{examples/authorization_details_sd_jwt_vc.json}}
+
+### Credential Request
+
+The following additional parameters are defined for Credential Requests and this Credential format.
+
+* `credential_definition`: REQUIRED. JSON object containing the detailed description of the credential type. It consists at least of the following sub claims:
+  * `type`: REQUIRED. JSON string as defined in (#authorization_vc_sd-jwt). The credential issued by the issuer MUST be of the type defined in this claim.
+  * `claims`: OPTIONAL. A JSON object as defined in (#authorization_vc_sd-jwt). This object determines the subset of authorized claims to be added to the credential to be issued.
+
+The following is a non-normative example of a Credential Request with Credential format `vc+sd-jwt`.
+
+<{{examples/credential_request_sd_jwt_vc.json}}
+
+### Credential Response {#credential_response_jwt_vc_json}
+
+The value of the `credential` claim in the Credential Response MUST be an SD-JWT VC. Credentials of this format are already suitable for transfer and, therefore, they need not and MUST NOT be re-encoded.
+
+The following is a non-normative example of a Credential Response with Credential format `vc+sd-jwt`.
+
+<{{examples/credential_response_sd_jwt_vc.txt}}
+
+### Verifier Metadata
+
+The Verifier SHOULD add a `vp_formats_supported` element to its metadata (e.g. in the `client_metadata` authorization request parameter) to let the wallet know what protection algorithms it supports in conjunction with SD-JWT VCs. The format element MUST have the key ``vc+sd-jwt`, the value is an objec consisting of the following elements:
+
+* `sd-jwt-alg`: OPTIONAL. A JSON array containing identifiers of cryptographic algorithms the verifier supports for protection of a SD-JWT. If present, the `alg` JOSE header (as defined in [@!RFC7515]) of the presented SD-JWT MUST match one of the array values.
+* `kb-jwt-alg`: OPTIONAL. A JSON array containing identifiers of cryptographic algorithms the verifier supports for protection of a KB-JWT. If present, the `alg` JOSE header (as defined in [@!RFC7515]) of the presented KB-JWT MUST match one of the array values.
+
+The following is a non-normative example of `client_metadata` request parameter value in a request to present a SD-JWT VC.
+
+<{{examples/client_metadata_sd_jwt_vc.json}}
+
+### Presentation Definition
+
+The presentation of a SD-JWT VC is requested by adding an object named `vc+sd-jwt` to the `format` object of an `input_descriptor`. The object is empty.
```

Comment:
Is there any integration (or plants to) with the DIF Claim format Registry? 

The Presentation Exchange specification links to this spec for a list of claim formats and their structure to use them within the formats property. 

https://identity.foundation/claim-format-registry/

Reply:
yes, it would be ideal if we can align with the PR spec.

Reply:
filed issue https://github.com/vcstuff/oid4vc-haip-sd-jwt-vc/issues/63